docs/api.rst

```diff
@@ -113,13 +112,21 @@ Adds new language to the library under the specified name. Used mostly internall
   to use common regular expressions defined within it.
 
 
+``registerAlias(alias|aliases, {languageName})``
+------------------------------------------------
+
+Adds new language alias or aliases to the library for the specified language name defined under ``languageName`` key.
+
+* ``alias|aliases``: a string or array with the name of alias being registered
+* ``languageName``: the language name as specified by ``registerLanguage``.
+
+
 ``listLanguages()``
-----------------------------
+-------------------
```

Comment:
Thanks for cleaning all these up, or is that a linter of some sort?

Reply:
No. it&rsquo;s just me.